src/data-access/repositories/mongo/mongo-status-message-repository.ts

```diff
@@ -30,6 +30,13 @@ export class MongoStatusMessageRepository extends BaseMongoRepository implements
     return await this.getCollection().find<StatusMessage>({}).toArray()
   }
 
+  public async getStatusMessagesWithIdPrefix(idPrefix: string): Promise<StatusMessage[]> {
+    this.assertDatabaseConnection(this.getStatusMessagesWithIdPrefix.name)
+    return await this.getCollection().find<StatusMessage>({
```

Comment:
await is redundant and can be removed. Same goes for `getAllStatusMessages`.

Reply:
Removed